index.bs

```diff
@@ -4629,75 +4618,88 @@ Note: If an error name is not listed here, please file a bug as indicated at the
             <td>The object can not be cloned.</td>
             <td><dfn id="dom-domexception-data_clone_err" for="DOMException" const export><code>DATA_CLONE_ERR</code></dfn> (25)</td>
         </tr>
         <tr>
-
             <td>"<dfn id="encodingerror" exception export><code>EncodingError</code></dfn>"</td>
             <td>The encoding operation (either encoded or decoding) failed.</td>
             <td>—</td>
         </tr>
         <tr>
-
             <td>"<dfn id="notreadableerror" exception export><code>NotReadableError</code></dfn>"</td>
             <td>The I/O read operation failed.</td>
             <td>—</td>
         </tr>
         <tr>
-
             <td>"<dfn id="unknownerror" exception export><code>UnknownError</code></dfn>"</td>
             <td>The operation failed for an unknown transient reason (e.g. out of memory).</td>
-
             <td>—</td>
         </tr>
         <tr>
-
             <td>"<dfn id="constrainterror" exception export><code>ConstraintError</code></dfn>"</td>
             <td>A mutation operation in a transaction failed because a constraint was not satisfied.</td>
             <td>—</td>
         </tr>
         <tr>
-
             <td>"<dfn id="dataerror" exception export><code>DataError</code></dfn>"</td>
             <td>Provided data is inadequate.</td>
             <td>—</td>
         </tr>
         <tr>
-
             <td>"<dfn id="transactioninactiveerror" exception export><code>TransactionInactiveError</code></dfn>"</td>
             <td>A request was placed against a transaction which is currently not active, or which is finished.</td>
             <td>—</td>
         </tr>
         <tr>
-
             <td>"<dfn id="readonlyerror" exception export><code>ReadOnlyError</code></dfn>"</td>
             <td>The mutating operation was attempted in a "readonly" transaction.</td>
             <td>—</td>
         </tr>
         <tr>
-
             <td>"<dfn id="versionerror" exception export><code>VersionError</code></dfn>"</td>
             <td>An attempt was made to open a database using a lower version than the existing version.</td>
             <td>—</td>
         </tr>
         <tr>
-
             <td>"<dfn id="operationerror" exception export><code>OperationError</code></dfn>"</td>
             <td>The operation failed for an operation-specific reason.</td>
             <td>—</td>
         </tr>
         <tr>
-
             <td>"<dfn id="notallowederror" exception export><code>NotAllowedError</code></dfn>"</td>
             <td>The request is not allowed by the user agent or the platform in the current context, possibly because the user denied permission.</td>
             <td>—</td>
         </tr>
     </tbody>
 </table>
 
+<div class=advisement>
+    Additionally, the following error names are kept for legacy reasons but their usage is discouraged:
+
+    <table id="legacy-error-names" class="vert data">
+        <thead>
+            <tr>
+                <th>Name</th>
```

Comment:
This contains more than just the name; it contains the legacy code name and value too. Maybe something more generic like "Error"